src/main/java/pb360/model/embedded/Line.java

```diff
@@ -0,0 +1,28 @@
+package pb360.model.embedded;
+
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class Line {
+	private String line;
```

Comment:
I think we should not create a model for containning one variable

Reply:
i thought i should use @JsonProperty for each line. I deleted this